source/Projectile.js

```diff
@@ -0,0 +1,23 @@
+var Pixi = require("pixi.js")
+var Reference = require("Reference.js")
+
+export default class Projectile extends Pixi.Sprite {
+    constructor(x, y) {
+        var projectileTex =
+            Pixi.Texture.fromImage(require("./images/projectile.png"))
+        super(projectileTex)
+        this.x = x
+        this.y = y
+        this.speed = 80
+        console.log("Creating projectile")
```

Comment:
Agreed. For now though, I intend to leave these in until the bullet collision mechanics are done in what I figure will be the pull request after next. Probably won't need it, but I hate rewriting code that I just deleted and I don't forsee a log of console clutter in the next few commits.